my-demo-project/calc/src/test/java/org/example/demo/calc/CalcTest.java

```diff
@@ -0,0 +1,30 @@
+package org.example.demo.calc;
+
+import org.junit.Test;
+
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class CalcTest {
+
+    @Test
+    public void averageMarkOfStudent() {
+        //given
+        List<Integer> integerList=List.of(1,2,3,4,5);
+        double expectedAverage = 3.0;
+        //when
+        double actualAverage = Calculator.averageMarkOfStudent(integerList);
+        //then
+        assertEquals(expectedAverage, actualAverage,0.0);
```

Comment:
Опять-таки, нужны пробелы между параметрами метода.
```suggestion
        assertEquals(expectedAverage, actualAverage, 0.0);
```